framework/di/Container.php

```diff
@@ -426,7 +426,9 @@ protected function getDependencies($class)
         $constructor = $reflection->getConstructor();
         if ($constructor !== null) {
             foreach ($constructor->getParameters() as $param) {
-                if ($param->isDefaultValueAvailable()) {
+                if ($param->isVariadic()) {
```

Comment:
`if(method_exists($param, 'isVaridic') && $param->isVariadic())`

works on php 5.4 and 5.5 we can stll have this for 2.0.12